proto/sources.proto

```diff
@@ -111,8 +111,11 @@ message ECR {
   repeated string registries = 2;
 }
 
-message Filesystem{
+message Filesystem {
+  // directories is deprecated and can be removed / renamed to paths when
+  // we no longer depend on the name in enterprise configs
   repeated string directories = 1;
+  repeated string paths = 2;
 }
```

Comment:
I'm not sure how to best handle this. We need to keep `directories` for backwards compatibility with enterprise configs (like how we keep `--directories` flag functionality in this PR).

We could technically just have `directories` and pass all paths there, but the name is misleading.

I decided to create a new entry for paths to be clear about the data it contains and we can remove directories in the future if/when config files aren't directly dependent on proto names.

Reply:
Suggestion: Add something like `// DEPRICATED` for stuff we want to remove in the next major release, so it is easy to grep for it since we probably won't remember when the time comes.